common/OpTestOpenBMC.py

```diff
@@ -447,22 +448,23 @@ def clear_sel(self):
     '''
     set boot device to setup
     curl -c cjar -b cjar -k -H "Content-Type: application/json" -d "{\"data\": \"Setup\"}" -X PUT
-    https://bmc/org/openbmc/settings/host0/attr/boot_flags
+    https://bmc/xyz/openbmc_project/control/host0/boot/attr/bootsource
     '''
     def set_bootdev_to_setup(self):
         data = '\'{"data": \"Setup\"}\''
-        obj = "/org/openbmc/settings/host0/attr/boot_flags"
+        obj = "xyz/openbmc_project/control/host0/boot/attr/bootsource"
```

Comment:
And here is there any way we can get the current bootdev setting using REST API, so that we can add new function here to actually verify whether boot device is really honoured or not, like we have added similar functions in ipmi case(Refere OpTestOOBIPMI.py test). And in OpenBMC docs still i see old settings/host0/boot_flags property here https://github.com/openbmc/docs/blob/master/host-management.md